src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs

```diff
@@ -46,15 +49,22 @@ public class TemplateController : Controller
             _notifier = notifier;
             S = stringLocalizer;
             H = htmlLocalizer;
+            _hca = hca;
         }
 
         public Task<IActionResult> Admin(PagerParameters pagerParameters)
         {
+            var query = "";
+            if (_hca.HttpContext.Request.Query.ContainsKey("DisplayText"))
+            {
+                query = _hca.HttpContext.Request.Query["DisplayText"];
+            }
```

Comment:
Ezt nem lehet úgy leírni egy sorban, hogy `null` legyen a változó, ha nincs ilyen paraméter? A `?` operátorral?

Amúgy ez nem `query`, inkább `displayText`.

Reply:
Kapok egy ArgumentNullException-t hogy a paraméter nem lehet null

Reply:
Az `Index()`-ben amúgy is kellene rá egy `string.IsNullOrEmpty()` check.

Reply:
Tévedtem, az más miatt volt, már okés